.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Looks like you need a special plugin for your text editor for this to work? Could add instructions to the readme explaining how?

Reply:
Yeah it does require an IDE extension to take effect. What do you think? Should we leave it our the repo for now and leave it up to devs if they want to add one? We could then ignore it in the .gitignore. I've added it to the standards doc in https://github.com/sanger/.github/pull/2/files so we could get some more input.

Reply:
Will leave it in until dissent.